pkg/validations/validation_engine.go

```diff
@@ -2,43 +2,79 @@ package validations
 
 import (
 	// Used to embed yamls by kube-linter
-	_ "embed"
+	"context"
+	_ "embed" // nolint:golint
 	"fmt"
 	"os"
-	"strings"
 
 	// Import checks from DVO
-	_ "github.com/app-sre/deployment-validation-operator/pkg/validations/all"
+	"github.com/app-sre/deployment-validation-operator/pkg/configmap"
+	_ "github.com/app-sre/deployment-validation-operator/pkg/validations/all" // nolint:golint
 
-	"golang.stackrox.io/kube-linter/pkg/builtinchecks"
 	"golang.stackrox.io/kube-linter/pkg/checkregistry"
 	"golang.stackrox.io/kube-linter/pkg/config"
 	"golang.stackrox.io/kube-linter/pkg/configresolver"
 	"golang.stackrox.io/kube-linter/pkg/diagnostic"
 
 	// Import and initialize all check templates from kube-linter
-	_ "golang.stackrox.io/kube-linter/pkg/templates/all"
+	_ "golang.stackrox.io/kube-linter/pkg/templates/all" // nolint:golint
 
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/spf13/viper"
 )
 
-var engine validationEngine
+var engine ValidationEngine
```

Comment:
Let's not do it in this PR, but it would be good to remove it in the future.

Reply:
You mean the "global variable" approach or the export of the structure?

Reply:
yes the global variable "hack" :)